CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Packaging
 
 - Added compatibility logo variants for environments which can't render the default SVG
+- Added /Applications symlink into OS X DMG for easier installation
```

Comment:
Packaging is usually about changes that affect packagers, I've thought about adding it there too, but I think the `Added` section makes more sense.

The though is right, I just don't think it's the best place in this particular PR, since it has no effect on packagers at all (brew doesn't use the dmg).

Reply:
Ah, alright. I thought It's about generic packaging and `Added` is more about functional changes.